client/src/hooks/useProductionModal.jsx

```diff
@@ -171,6 +174,18 @@ const useProductionModal = () => {
           note: noteRef.current.value,
         });
       }
+      // update scheduling page
+      await createScheduling({
+        partType: nameRef.current.value,
```

Comment:
somethings wrong with this, 
![image](https://user-images.githubusercontent.com/43047923/114643489-bd08b980-9ca3-11eb-992d-b71bd3fd04af.png)

Reply:
nevermind